### DISCUSSION TITLE
Why is version codename is my president's name?

### DISCUSSION THREAD

**Discussion by PackmanDude** [Pryaxis/TShock#2654] [General]
![зображення](https://user-images.githubusercontent.com/99031872/184506952-dad1438e-8175-4264-b90f-b1775646035a.png)

Looks sus.

**Comment by hakusaro** [comment]
![244DD2FA-F3A5-409A-BE3F-A523B092AA26](https://user-images.githubusercontent.com/532647/184508679-bc5b60b6-d34c-4a5f-b405-a3dca7ddbba7.jpeg)

**Comment by hakusaro** [comment]
We have changed the codename in version 4.5.18.
